polyfill/lib/absolute.mjs

```diff
@@ -0,0 +1,138 @@
+import { ES } from "./ecmascript.mjs";
+import { SLOT_EPOCHNANOSECONDS } from "./slots.mjs";
+
+export function Absolute(epochNanoSeconds) {
+  if (!(this instanceof Absolute)) return new Absolute(epochNanoseconds);
```

Comment:
why not use class syntax for these?

Reply:
Because most other intrinsic objects can be created/cast while omitting the `new`. For consistency sake, I wanted the same behaviour. In additions `Time(15, 30, 45)` without the new actually feels more natural/convenient, especially given that they are immutable objects.

Reply:
ES6 and later tends to require `new` for classes, rather than making them callable. I think we should follow this convention in Temporal.

Reply:
I disagree because of patterns like:

dateTime.with({ day: 5 })
dateTime.with(Date(2019,8,10))
Time(dateTime) // casting to Time
Date(dateTime) // casting to Date

I think this makes things so much more ergonomic. So if you feel strongly, can you open an issue so we can have that discussion properly.

Reply:
Agreed with @ljharb. Let's go over these sorts of decisions in the Temporal meeting today; I put it on the agenda.